Exercises/Java Scrip/Ex_4/index.js

```diff
@@ -0,0 +1,5 @@
+alert("Welcome!");
+let firstNumber = prompt("Enter First Number");
+let secondNumber = prompt("Enter Second Number");
```

Comment:
Here would be better to convert entered values to "number" type and then compare them.